common/define-grammar.js

```diff
@@ -96,6 +112,18 @@ module.exports = function defineGrammar(dialect) {
       $._namespace_use_type,
     ],
 
+    reserved: {
+      global: _ => BASE_RESERVED_KEYWORD_SET.concat([keyword('static', false), keyword('var', false)]),
+      // 'var' is allowed in variable names
+      variables: _ => BASE_RESERVED_KEYWORD_SET.concat([keyword('static', false)]),
```

Comment:
This should be empty like properties, all of these keywords are allowed as variables